### DISCUSSION TITLE
[Kontoverwaltung] Übersicht über Lehrer als Admin

### DISCUSSION THREAD

**Discussion by Schmarvinius** [SE-TINF22B2/G2-Schoolify#130] [User Stories]
## User Story:
Als Admin will ich eine Übersicht über einen Lehrer haben. 

## Beschreibung
Eine Ansicht in der persönliche Daten angezeigt werden. Dazu sollen alle aktuell unterrichtenden Fächer + Klasse angezeigt werden.

## Akzeptanz-Kriterien
- [ ] Persönliche Informationen zu Lehrer sollen dargestellt werden
- [ ] Liste aller aktuell unterrichtenden Klassen soll angezeigt werden

### Issue
- #144 
- #145

**Comment by laralngn** [comment]
Nutzen der User Story fehlt. Warum will ein Admin diese Übersicht?

**Reply by laralngn** [reply]
Fehlt immer noch

**Reply by laralngn** [reply]
Fehlt immer noch :(

**Comment by laralngn** [comment]
Wenn die User Story und ihre Issues beibehalten werden, muss die Kernfunktion angeglichen werden. Entweder bei allem Kontoverwaltung oder bei allem Informationssystem.
